src/resolvers/users.js

```diff
@@ -71,6 +72,48 @@ const usersResolvers = {
       await user.save();
       return user;
     },
+
+    async verifyAccount(root, { id, hash }) {
+      const user = await models.User.findByPk(id);
+      if (user == null) {
+        throw new UserInputError('User not found');
+      }
+
+      if (user.isVerified) {
+        return user;
+      }
+
+      if (user.verificationHash === hash) {
+        user.isVerified = true;
+        user.verificationHash = null;
```

Comment:
curious, is this necessary/desired?

Reply:
I think it's not necessary, but it is desired. Since we no longer need the hash, we should delete it from our database. 

Check out: https://stackoverflow.com/questions/39092822/how-to-confirm-email-address-using-express-node

Reply:
Thanks for the explanation!